oracle/oracle.go

```diff
@@ -91,32 +96,37 @@ func NewClient() (cloudops.Ops, error) {
 
 func getInfoFromEnv(oracleOps *oracleOps) error {
 	var err error
-	oracleOps.instance, err = cloudops.GetEnvValueStrict(envInstanceID)
+	oracleOps.instance, err = cloudops.GetEnvValueStrict(envPrefix + "_" + envInstanceID)
```

Comment:
envPrefix is not required. change it everywhere.

Reply:
Done